Slambook/index.js

```diff
@@ -54,30 +55,8 @@ form.addEventListener("submit", (e) => {
```

Comment:
To set the class attribute of the created element, use 'element.classList.add' method instead of directly assigning to 'element.classList'.